data/core/units/undead/Corpse_Walking.cfg

```diff
@@ -29,11 +29,12 @@
 #define UNIT_BODY_WALKING_CORPSE_GRAPHICS BASE_NAME
     image="units/undead/{BASE_NAME}.png"
     {DEFENSE_ANIM "units/undead/{BASE_NAME}-defend.png" "units/undead/{BASE_NAME}.png" {SOUND_LIST:ZOMBIE_WEAK_HIT} }
-    # This standing animation only exists for default zombie for now. As variations are added, the filter should be updated
+    # This standing animation only exists for the base unit and variation="mounted" for now. As variations are added, the two filters should be updated.
     [standing_anim]
         [filter]
+            #included are the base unit and variation="mounted"
```

Comment:
Why is this done via `[not]`? No way to refer to the base unit?

Reply:
Hi, yes, exactly. The logic with [not] was already there, and I have also wondered if there is a better way. But at least as far as I know, specifying the base unit doesn't seem to be possible in WML. Fortunately, this way works (if the filter is correct).

Reply:
I guess `variation=` without a `[not]` does not work? Or maybe `variation=""`?

Reply:
Hi, I have now tried your ideas. Unfortunately, both don't seem to work. But I don't think that's a big deal, the version with [not] is perhaps not extremely elegant, but it works.

Reply:
Well, the issue is maintainability. As this PR shows it's likely that this filter will fall out of date again.

There is already issue #5498 though. So the filter can be improved once that is resolved.

Reply:
Thanks, this issue / feature request fits very well.

The current errors in the filters all date back to the filters' original creation 16 days ago in b0d327e (#7102). I hope that simply updating the filters when a new animation is added is easier, since a new animation will probably also be tested.

But I see your point that this could be a potential problem.